packages/cli-tool/src/helpers/child-process.ts

```diff
@@ -0,0 +1,18 @@
+import {spawn} from 'node:child_process'
+
+const runCommand = (name:string, options: string[]): Promise<boolean> => {
+  const initializePromise = new Promise((resolve:(value: boolean)=>void, reject) => {
```

Comment:
Can we add some spaces here?

```suggestion
const runCommand = (name: string, options: string[]): Promise<boolean> => {
  const initializePromise = new Promise((resolve: (value: boolean) => void, reject) => {
```

Reply:
All spacing issues fixed in: 7269e17 ;-)